Makefile

```diff
@@ -1,9 +1,12 @@
 RUNC_TEST_IMAGE=runc_test
 PROJECT=github.com/opencontainers/runc
 TEST_DOCKERFILE=script/test_Dockerfile
-BUILDTAGS=seccomp
 export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
 
+# allow for passing buildtags to make or setting the default
+DEFAULT_BUILDTAGS=seccomp
+BUILDTAGS := $(if $(BUILDTAGS),$(BUILDTAGS),$(DEFAULT_BUILDTAGS))
```

Comment:
i did try but maybe it was me doing it wrong haa

On Fri, Jan 15, 2016 at 4:56 PM, Tianon Gravi <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/opencontainers/runc/pull/482#discussion_r49922856>:
>
> >  export GOPATH:=$(CURDIR)/Godeps/_workspace:$(GOPATH)
> >
> > +# allow for passing buildtags to make or setting the default
> > +DEFAULT_BUILDTAGS=seccomp
> > +BUILDTAGS := $(if $(BUILDTAGS),$(BUILDTAGS),$(DEFAULT_BUILDTAGS))
>
> Uhh, this should already work without doing this -- Make lets you override
> variables on the command line directly like make BUILDTAGS='a b c' ...;
> is it not working?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runc/pull/482/files#r49922856>.
>